backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/LookupController.cs

```diff
@@ -65,7 +66,6 @@ public async Task<ActionResult<Party>> GetOrganisation(string orgNummer)
         /// <param name="partyId">The partyId for the reportee to look up</param>
         /// <returns>Reportee if party is in authenticated users reporteelist</returns>
         [HttpGet]
-        [Authorize]
         [Route("accessmanagement/api/v1/lookup/reportee/{partyId}")]
```

Comment:
Route needs to be updated for all endpoints since [Route("accessmanagement/api/v1/lookup")] is now set on controller level

Reply:
Reverted file